wearos_app/build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+plugins {
+    id("com.android.application")
+    id("kotlin-android")
+    id("kotlin-kapt")
+    id("kotlin-android-extensions")
+}
+
+android {
+    compileSdkVersion(Config.Android.compileSdk)
+
+    defaultConfig {
+        applicationId = "io.homeassistant.companion.wearos"
```

Comment:
This I believe needs to be the same as the android app.  Last line of this section: https://developer.android.com/training/wearables/apps/packaging#specifying-app-as-standalone